scripts/.script_env

```diff
@@ -2,7 +2,7 @@
 # shellcheck disable=SC2034
 set -o pipefail -o errexit
 
-VAULT_FOLDER=web-vault
+VAULT_FOLDER=${VAULT_FOLDER:-web-vault}
```

Comment:
I think it's better to use `:=` instead of `:-`.
Because `:=` will also use the default value if it's set to an empty value for example.